.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+    "env": {
+        "browser": true,
+        "node": false
+    },
+    "parser": "babel-eslint",
+    "extends": ["eslint:recommended", "standard"],
+    "rules": {
+        "no-console": 0,
```

Comment:
unfortunately we have to leave the no-console rule set off globally because it's part of the application flow. 
I'd like to improve this later.